parsing/builtin_attributes.ml

```diff
@@ -92,15 +92,17 @@ let builtin_attrs =
 
 let is_builtin_attr s = Hashtbl.mem builtin_attrs s
 
+let track = ref true
```

Comment:
Right around here would be a great spot for that Note.

Even if we decide not to do the full glorious Note (but I think we should!), please comment this state.